components/modals/AppRouterEmailForm.tsx

```diff
@@ -26,12 +35,23 @@ export const EmailForm = (props: EmailFormProps) => {
     setIsProcessing(true);
     setMessage({ text: '', type: '' });
     try {
-      const result = await addToMailchimp(email, firstName, lastName);
+      const result = await addToMailchimp(
+        formData.email,
+        formData.firstName,
+        formData.lastName,
+        formData.notes,
+      );
       if (result.result === 'success') {
         setMessage({
           text: "You've been added to the llama list.",
           type: 'success',
         });
+        setFormData({
+          firstName: '',
+          lastName: '',
+          email: '',
+          notes: '',
+        });
```

Comment:
Now we can re-use the default values
```suggestion
        setFormData(defaultValues.current);
```